### PR TITLE
added more information from previous commit

### PR DESCRIPTION
**Pull request by bfallonf**
Added information from a previous commit to the Origin docs by @sosiouxme : 

https://github.com/openshift/origin/commit/1cd305e1ff1439a7b5be2d70cfbd0e748787459b

Brought to my attention by @adellape . Asking for a peer review, please!

### CODE REVIEW

File: architecture/openshift_model.adoc [hunk 1]
```diff
@@ -39,10 +39,13 @@ Source-to-image (sti) is a tool for building reproducible Docker images. It prod
 The custom build strategy is very similar to *Docker build* strategy, but users might customize the builder image that will be used for build execution. The Docker build uses https://registry.hub.docker.com/u/openshift/docker-builder/[openshift/docker-builder] image by default. Using your own builder image allows you to customize your build process.
 
 == BuildLog
+A buildlog is a model object that stores the logs of a particular build for later inspection.
 
 == Deployment
+A deployment is a specially annotated replication controller, specifying the desired configuration of that controller. See the https://github.com/openshift/origin/blob/master/docs/deployments.md[deployment documentation] for more information.
```
Comment (sosiouxme) [inline architecture/openshift_model.adoc]: I assume the long-term plan is not to leave this pointing at dev docs :)

FWIW, I think the idea of deployments is probably stable enough to begin documenting if you want to bring that over to openshift-docs so you have a reference for this to point at.

File: architecture/openshift_model.adoc [hunk 2]
```diff
@@ -317,9 +320,7 @@ Sample Template:
 ----
 
 == Route
-An OpenShift route is a way to announce your service to the world.  A route, consumed by a router in conjunction with
- service endpoints provides named connectivity from external sources to your applications.  Each route provides a name, service
- selector, and optionally security configuration.
+An OpenShift route is a way to announce your service to the world.  A route, consumed by a router in conjunction with service endpoints provides named connectivity from external sources to your applications.  Each route provides a name, service selector, and optionally security configuration.
```
Comment (sosiouxme) [inline architecture/openshift_model.adoc]: Maybe add a comma: "A route, consumed by a router in conjunction with service endpoints, provides named connectivity from external sources to your applications.

**Comment by sosiouxme** [comment]
I see you've copied over the stuff I whipped up just to have something. Well, that will do for now :)

BTW, can I just say how much better it is to review a PR here than to do this in email!

File: architecture/openshift_model.adoc (hunk 1 above)
Comment (adellape) [inline architecture/openshift_model.adoc]: That info on deployments was brought over recently to ../using_openshift/deployments.adoc:

https://ci.openshift.redhat.com/openshift-docs-master-testing/latest/using_openshift/deployments.html

Can probably point to that instead for those links in "Deployment" and "DeploymentConfig".

Comment (bfallonf) [inline architecture/openshift_model.adoc]: Done. Links now point to http://docs.openshift.org/latest/using_openshift/deployments.html

**Comment by bfallonf** [comment]
@sosiouxme yes, the plan (for Beta, at least) is to fill in the gaps at the minimum. 

And yes, agreed. Now I've started working in Github, I can definitely  see the advantages :)

**Comment by sosiouxme** [comment]
@bfallonf that's it for now. I'm sure I'll have more to add when I'm more familiar with v3.

**Comment by nhr** [comment]
@bfallonf please rebase and push again so that I can merge.

**Comment by sosiouxme** [comment]
And now you get to learn how much fun merge conflicts are :)

**Pull request closed without merging**